sass-style-guide.md

```diff
@@ -59,87 +42,111 @@ You should order your rules from generic to specific. Example:
 
 // Abstract Component Imports
 //---------------------------
-@import abstract-components/section-box
+@import abstract-components/boundary-box
+@import abstract-components/gradient-overlay
 
 // Generic Component Imports
 //--------------------------
 @import generic-components/button
-@import generic-components/skip-links
+@import generic-components/data-table
 
 // Specific Component Imports
 //---------------------------
-@import specific-components/page/page-footer
-@import specific-components/page/page-header
+@import specific-components/company-search
+@import specific-components/page-header
+```
+
+## Style-Definitionen
+
+Wann wird wo was definiert? Wir gehen inhaltsgetrieben vor, d.h. wir schreiben unseren Sass-Code in der Reihenfolge, wie die Elemente auf der gerenderten Seite erscheinen werden:
+
+1. **von außen nach innen**: ZASAF folgend beginnt man in der Regel mit dem Page-Wrapper-Element.
```

Comment:
Aber das steht ja nur in einer gesonderten Page-Komponente und es trifft nicht mehr auf spezifische Komponenten zu. Aber okay, man versteht das Prinzip.